src/web/mjs/connectors/WorldRomanceTranslation.mjs

```diff
@@ -1,13 +1,26 @@
 import WordPressMangastream from './templates/WordPressMangastream.mjs';
 
 export default class WorldRomanceTranslation extends WordPressMangastream {
-
     constructor() {
         super();
         super.id = 'worldromancetranslation';
         super.label = 'World Romance Translation';
-        this.tags = [ 'manga', 'indonesian' ];
+        this.tags = ['manga', 'indonesian'];
         this.url = 'https://wrt.my.id';
         this.path = '/manga/list-mode/';
+
+        this.queryChapters = 'div#chapterlist ul li div.eph-num';
+    }
+
+    async _getChapters(manga) {
```

Comment:
Is it possible to define a chapter query that does not require to overwriting the method?
e.g.
```javascript
this.queryChapters = 'div#chapterlist ul li div.eph-num a:last-of-type';
```

Reply:
I wasn't really familiar with pseudo selectors so I didn't think of this. Your solution works like a charm, so I'll keep it in mind for future fixes.